perun-core/src/test/java/cz/metacentrum/perun/core/entry/VosManagerEntryIntegrationTest.java

```diff
@@ -718,6 +720,62 @@ public void getVoMembersCountsByStatusWhenVoNotExists() throws Exception {
 		vosManagerEntry.getVoMembersCountsByStatus(sess, new Vo());
 	}
 
+	@Test
+	public void getMemberVos() throws Exception {
+		System.out.println(CLASS_NAME + "getMemberVos");
+
+		Vo vo = perun.getVosManagerBl().createVo(sess, myVo);
+		Vo memberVo = new Vo(-1, "Vo2", "vo2");
+		memberVo = perun.getVosManagerBl().createVo(sess, memberVo);
+		vosManagerEntry.addMemberVo(sess, vo, memberVo);
+
+		assertThat(vosManagerEntry.getMemberVos(sess, vo.getId())).containsExactly(memberVo);
+		assertThat(vosManagerEntry.getMemberVos(sess, memberVo.getId())).containsExactly();
+	}
+
+	@Test
+	public void getParentVos() throws Exception {
+		System.out.println(CLASS_NAME + "getParentVos");
+
+		Vo vo = perun.getVosManagerBl().createVo(sess, myVo);
+		Vo memberVo = new Vo(-1, "Vo2", "vo2");
+		memberVo = perun.getVosManagerBl().createVo(sess, memberVo);
+		vosManagerEntry.addMemberVo(sess, vo, memberVo);
+
+		assertThat(vosManagerEntry.getParentVos(sess, vo.getId())).containsExactly();
+		assertThat(vosManagerEntry.getParentVos(sess, memberVo.getId())).containsExactly(vo);
+	}
+
+	@Test
+	public void getParentVosNoRelations() throws Exception {
+		System.out.println(CLASS_NAME + "getParentVosNoRelations");
+
+		Vo vo = perun.getVosManagerBl().createVo(sess, myVo);
+		assertThat(vosManagerEntry.getParentVos(sess, vo.getId())).isEmpty();
+	}
+
+	@Test
+	public void removeNonexistingMemberVo() throws Exception {
```

Comment:
Maybe a test that covers normal behaviour of removeMemberVo() (not only the exception) would be nice to have.

Reply:
Thanks, good point. Added test and fixed javadocs.